### PR TITLE
some text tweaks

### PR DESCRIPTION
**Pull request by henricasanova**

### CODE REVIEW

File: paper_io_sim.tex [hunk 1]
```diff
@@ -378,7 +378,7 @@
     and append it to the active list.
     If the blocks are dirty, we move them independently to the active list, to preserve their entry time.
     Because the chunk and block sizes may be different, there are situations
-    where a block is not entirely read.
+    in a block is not entirely read.
```
Comment (glatard) [inline paper_io_sim.tex:381]: do you mean `in which`?

File: paper_io_sim.tex [hunk 2]
```diff
@@ -321,18 +321,18 @@
 
     The Memory Manager simulates two parallel threads: the main one
     implements flushing, eviction, and cached I/Os synchronously, whereas
-    the second one, operating in the background, periodically searches for
-    expired dirty data in page cache LRU lists and flushes them to disk. We
+    the second one, which operates in the background, periodically searches for
+    expired dirty data in page cache LRU lists and flushes data to disk. We
```
Comment (glatard) [inline paper_io_sim.tex:325]: do you mean `this data`?

File: paper_io_sim.tex (hunk 1 above)
Comment (henricasanova) [reply paper_io_sim.tex:381]: I thought ok saying "in which" but then thought "were" was ok. So this is a typo.

File: paper_io_sim.tex (hunk 2 above)
Comment (henricasanova) [reply paper_io_sim.tex:325]: yes. "this data".

**Pull request merged**